### DISCUSSION TITLE
DataTable template header when using expandableRows colspan wrong

### DISCUSSION THREAD

**Discussion by lucassmsilva** [primefaces/community#702] [PrimeVue]
When using expandableRows props, there's two issues that happen.
First, the row of the group doesnt have full width, it's always it a less colspan
Second, the row isnt flex so its harder to align it.
Below, the print and code example

![image](https://github.com/primefaces/community/assets/56162808/823f4036-3b81-4772-aa17-4cc2623cef0e)

When inspecting the row in the browser, will see that the colspan is wrong
![image](https://github.com/primefaces/community/assets/56162808/c41f388a-031b-4c72-a64a-93827cce4569)

```
<script setup>
import { ref } from 'vue';

const value = [
  {
    "name": "Sarah Miller",
    "age": 25,
    "email": "sarah.miller@email.com",
    "birthdate": "1998-05-12",
    "project": "Marketing"
  },
  {
    "name": "John Smith",
    "age": 32,
    "email": "john.smith@email.com",
    "birthdate": "1991-10-27",
    "project": "Design"
  },
  {
    "name": "Emily Jones",
    "age": 28,
    "email": "emily.jones@email.com",
    "birthdate": "1995-02-01",
    "project": "Development"
  },
  {
    "name": "Michael Lee",
    "age": 40,
    "email": "michael.lee@email.com",
    "birthdate": "1983-08-15",
    "project": "Design"
  },
  {
    "name": "Jessica Garcia",
    "age": 22,
    "email": "jessica.garcia@email.com",
    "birthdate": "2001-03-07",
    "project": "Marketing"
  },
];

const expandedRowGroups=ref();

</script>

<template>   

    <div class="w-full bg-blue-500">
        width full
    </div>


    <DataTable :value="value" 
        class="p-datatable-sm" 
        v-model:expandedRowGroups="expandedRowGroups"
        expandableRowGroups 
        rowGroupMode="subheader" groupRowsBy="project" sortMode="single"
        sortField="project" :sortOrder="1" scrollable scrollHeight="400px"
        tableStyle="min-width: 50rem"
        >
        
        <Column width="25%" field="name"></Column>
        <Column width="25%" field="age"></Column>
        <Column width="25%" field="email"></Column>
        <Column width="25%" field="birthdate"></Column>
        
        
        <template #groupheader="slotProps">
            <div class="flex flex-row w-full bg-red-500">
                {{ slotProps.data.project }}
            </div>
        </template>
</DataTable>
</template>
```

**Comment by tugcekucukoglu** [comment]
I've created a demo https://stackblitz.com/edit/zgmug5?file=src%2FApp.vue%3AL10 and cannot replicate the issue. Could you share a stackblitz link?

**Comment by WashingtonOlmos** [comment]
Hi, I have exactly the same problem, were you able to solve it? Could you share your solution with me? 
The arrows show where the background color should be, but it doesn't.
![image](https://github.com/user-attachments/assets/a42de9af-6f35-4ed6-ab42-f08bbbe35cd3)

my DataTable Configuration

![image](https://github.com/user-attachments/assets/a786c5d8-0f49-4560-b1e1-60b6538a8235)

**Comment by WashingtonOlmos** [comment]
I solved it by adding a column
` <Column field="categoria.descripcion" header="Categoria"></Column>
`
